contracts/gas-snapshots/vrf.gas-snapshot

```diff
@@ -0,0 +1,130 @@
+BatchVRFCoordinatorV2PlusTest:test_fulfillRandomWords() (gas: 1068535)
```

Comment:
This is a sign something is going wrong :)

Reply:
I could not explain to you why random tests in other foundry profiles fail the snapshot but I needed to do it to get the CI to stop complaining. Is there something im missing?

Reply:
vrf doesn't use the snapshot so I don't know how/why you generated it, but I don't think CI would ask for it? If it does please reach out top people instead of committing unrelated files

Reply:
I have no idea. All I know is that every time I commit something there's a 50/50 chance that some other profile's snapshot tests will stop passing until I update the thing. I figured it was just easier.